packages/web-components/src/components/cbp-list/cbp-list.stories.tsx

```diff
@@ -0,0 +1,104 @@
+export default {
+    title: 'Components/Simple List',
+    tags: ['autodocs'],
+    argTypes: {
+        tag: {
+          control: 'select',
+          description: 'Type of List',
+          options: ['ul', 'ol'],
+        },
+        context : {
+        control: 'select',
+        options: [ "light-inverts", "light-always", "dark-inverts", "dark-always"]
+        },
+        sx: {
+        description: 'Supports adding inline styles as an object of key-value pairs comprised of CSS properties and values. Values should reference design tokens when possible.',
+        control: 'object',
+        },
+    },   
+    args: {
+      tag: 'ul',
+    },
+};
+
+function generateItems(items) {
+    const html = items.map(({ content, sublist}) => {
+      if(sublist){
+        return `${content}`;
+      }else{
+        return `<li>${content}</li>`;      
+      }
+    });
+    return html.join('');
+  }
+  
+const Template = ({listItems, tag, context }) => {
+    return ` 
+    <cbp-list
+      ${tag ? `tag=${tag}` : ''}
+      ${context && context != 'light-inverts' ? `context=${context}` : ''}
+    >
+        ${generateItems(listItems)}
+    </cbp-list>
+    `;
+};
+
+export const SimpleList = Template.bind({});
+    SimpleList.args = {
+        listItems: [
+            {
+              content: "List Item 1",
+            },
+            {
+              content: "List Item 2",
+            },
+            {
+              content: "List Item 3",
+            },
+            {
+              content: "List Item 4",
+            },
+           
+            {//example of ul sublist
+              content: `
+                <ul>
+                  <li>List Sub-Item 1</li>
+                  <li>List Sub-Item 2</li>
+                  <li>List Sub-Item 3</li>
+                  <li>List Sub-Item 4</li>
+                  <li>List Sub-Item 5</li>
+
+                </ul>
+                `,
+              sublist: true
+            }, {
+              content: "List Item 5",
+            },
+            {//example of ol sublist
+              content: `
+                <ol>
+                  <li>List Sub-Item 1</li>
+                  <li>List Sub-Item 2</li>
+                  <li>List Sub-Item 3</li>
+                  <li>List Sub-Item 4</li>
+                  <li>List Sub-Item 5</li>
+                  <ol>
```

Comment:
Like in the CSS, this is wrong. Nested lists need to be inside of list items. They cannot be siblings.